core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -1401,7 +1406,7 @@ class ReplicaManager(val config: KafkaConfig,
 
     try {
       // First stop fetchers for all the partitions
```

Comment:
Update comment?  It's not actually stopping the fetchers here anymore.

Reply:
Makes sense, and I'll remove this comment.